practice/1_python_part_1/task3.py

```diff
@@ -16,4 +16,22 @@
 
 
 def build_from_unique_words(*lines: Iterable[str], word_number: int) -> str:
-    ...
+  words = []
+  final = ''
+
+  for line in lines:
+    unique =[]
+    temp = line.split()
+
+    for i in temp:
```

Comment:
You can use the `set` function to convert a list to a set. It has (roughly) the same performance but looks simpler:
```
unique = set(temp)
```